Split config files (#243)
vmware/vmware-aria-operations-integration-sdk: docs/troubleshooting_and_faq/other.md

- Check that the Cloud Proxy supports containerized adapters. Containerized adapter
  support is supported in VMware Aria Operations version 8.10.0 and later.

> ### Migrating connection-related elements from config.json to connections.json?

Reviewer: ```suggestion
### How do I migrate connection-related elements from config.json to connections.json?
```

As of version 1.0.0,

> we've removed all connection-related elements from the [project config file](../references/project_config.md)
> and migrated them to a new
> [project connections JSON file](../references/project_connections_config.md)(`connections.json`).
> As part of this change, both `mp-test` and `mp-build` will offer to migrate connection-related
> elements to the `connections.json` file when present in the `config.json` file
> (The new `connections.json` file is also added to the project's `.gitignore`
> to prevent sensitive information from being committed).
> Regardless of whether the user decides to migrate the values or not, the found elements will be deleted from the
> `configuration.json` file.
> Moving all the connection-related information away from the `config.json` file allows users
> to include their project configuration file in version control,
> making using the same `container_repository` for the project easier.

Reviewer: ```suggestion
all connection-related elements from the [project config file](../references/project_config.md)
have been migrated to a new
[project connections JSON file](../references/project_connections_config.md)(`connections.json`).
As part of this change, both `mp-test` and `mp-build` will offer to migrate connection-related
elements to the `connections.json` file when present in the `config.json` file
(The new `connections.json` file is automatically added to the project's `.gitignore`
to prevent sensitive information from being committed).
Moving all the connection-related information away from the `config.json` file allows users
to include their project configuration file in version control,
making using the same `container_repository` for the project easier.
```
